obspy/io/ascii/tests/data/tspair_float_custom_fmt.ascii

```diff
@@ -1,13 +1,13 @@
-TIMESERIES XX_TEST__BHZ_R, 12 samples, 40 sps, 2008-01-15T00:00:00.025000, TSPAIR, FLOAT, Counts
-2008-01-15T00:00:00.025000  +185.00999
-2008-01-15T00:00:00.050000  +181.02
-2008-01-15T00:00:00.075000  +185.03
-2008-01-15T00:00:00.100000  +189.03999
-2008-01-15T00:00:00.125000  +194.05
-2008-01-15T00:00:00.150000  +205.06
-2008-01-15T00:00:00.175000  +209.07001
-2008-01-15T00:00:00.200000  +214.08
-2008-01-15T00:00:00.225000  +222.09
-2008-01-15T00:00:00.250000  +225.98
-2008-01-15T00:00:00.275000  +226.99001
-2008-01-15T00:00:00.300000  +219
+TIMESERIES XX_TEST__BHZ_R, 12 samples, 40 sps, 2008-01-15T00:00:00.025000, TSPAIR, CUSTOM, Counts
```

Comment:
See above comment, I think it's really strange to specify `CUSTOM`, when the string formatting clearly is of float type..